doc/design/object-bucket-lib.md

```diff
@@ -268,16 +281,19 @@ metadata:
   - name: MY-BUCKET-1 [5]
     ...
 type: Opaque
-data: [6]
+data:
   ACCESS_KEY_ID: BASE64_ENCODED-1
   SECRET_ACCESS_KEY: BASE64_ENCODED-2
+  ... [6]
 ```
 1. same name as the OBC. Unique since the secret is in the same namespace as the OBC.
 1. namespce of the originating OBC.
-1. label may be used to associate all artifacts under a paeticular provisioner.
+1. label may be used to associate all artifacts under a particular provisioner.
 1. finalizers set and cleared by the lib's OBC controller. Prevents accidental deletion of the Secret.
 1. ownerReference makes this secret a child of the originating OBC for clean up purposes.
-1. **Note:** the library will create the Secret using `stringData:` and let the Secret API base64 encode the values.
+1. ACCESS_KEY_ID and SECRET_ACCESS_KEY are the only secret keys defined by the library.
+Provisioners are able to cause the lib to create additional keys by returning  the `AdditionalConfig` field.
```

Comment:
field name changed to `AdditionalSecretData`.  Any values defined here are written to the secret directly.  It may be confusing to say they result in additional keys being created.